src/main/java/uk/gov/hmcts/reform/prl/services/SendgridService.java

```diff
@@ -53,6 +53,7 @@ public class SendgridService {
     public static final String PRL_RPA_NOTIFICATION = "Private Reform Law CCD Notification ";
     public static final String MAIL_SEND = "mail/send";
     public static final String CASE_NAME = "caseName";
+    public static final String CASE_NUMBER = "caseNumber";
```

Comment:
no need of this. use it from PrlAppsConstants

Reply:
Resolved